### DISCUSSION TITLE
[Show and Tell] <Triple RPS Boost Update to Maximise dApp Performance in the Bull Market>

### DISCUSSION THREAD

**Discussion by JoGetBlock** [zkSync-Community-Hub/zksync-developers#239] [Show and Tell]
### Project Name

GetBlock

### Repo Link

https://getblock.io/nodes/zksync/

### Technologies Used

RPC API

### Describe your project. What does it do? Why did you create it? Any interesting challenges you faced?

GetBlock is a project developed to harness the full potential of the ZkSync layer. Our mission with GetBlock is to simplify and streamline access to blockchain data for developers and businesses operating within the ZkSync ecosystem.

In easy words, GetBlock is a blockchain data service that provides fast and easy access to various blockchain data on the ZkSync network (and other 50+ chains). GetBlock allows developers to effortlessly query the data they need without the complexity of running their own ZkSync nodes. This includes transaction histories, wallet balances, smart contract interactions, and more.

Our journey began with a simple realization: accessing blockchain data can be daunting, especially on newer, less established layers like ZkSync. We noticed that many developers and small enterprises were struggling to leverage the ZkSync network's full capabilities due to the technical challenges and resources required to maintain nodes. GetBlock was created to bridge this gap, providing a straightforward and efficient solution to access blockchain data.

### What specific feedback are you looking for?

Any Feedback from users or projects that might have used or are about to use our RPC endpoints. We are very client-oriented, flexible and we would like to improve our service thanks to the feedback that  we receive from our clients. 

### How can the community help or contribute to your project?

Share your Opinion about GetBlock.io. Tell us about your suggestions or simply refer the service to relevant projects to evolve the ZkSync ecosystem. 

### Additional Information (Optional)

Please be informed of a technical update on [GetBlock.io](http://getblock.io/) RPC Node Provider. The rate limit for all shared plans of ZkSync Nodes will be increased from 60 to 200 requests/second (RPS). This update is part of ongoing service enhancements.

**Comment by dougtende** [comment]
Congratulations on the technical update to increase the rate limit for ZkSync Nodes on GetBlock.io from 60 to 200 requests per second (RPS)! This enhancement will undoubtedly improve user experience and facilitate smoother access to blockchain data for developers and businesses operating within the ZkSync ecosystem. Keep up the great work!

**Comment by go4comdathern** [comment]
GetBlock.io provides RPC API endpoints for accessing blockchain data on the ZkSync network and over 50 other chains. Our aim is to simplify blockchain data access for developers and businesses, enabling them to query transaction histories, wallet balances, smart contract interactions, and more without the hassle of running their own nodes. We welcome feedback from users and projects to enhance our service and contribute to the evolution of the ZkSync ecosystem. Additionally, we've increased the rate limit for shared plans of ZkSync Nodes from 60 to 200 requests/second to better serve our clients' needs.

**Comment by springsj** [comment]
GetBlock is a blockchain data service providing easy access to ZkSync and over 50 other chains. We welcome feedback from users and projects to enhance our service and contribute to the ZkSync ecosystem.

**Comment by inutpediflinda** [comment]
GetBlock is a blockchain data service providing fast and easy access to ZkSync network data via RPC API. Seeking feedback and suggestions for further improvement.

**Comment by rafordtetseyd** [comment]
GetBlock is a blockchain data service that provides fast and easy access to various blockchain data on the ZkSync network (and other 50+ chains) through RPC API. They aim to simplify and streamline access to blockchain data for developers and businesses operating within the ZkSync ecosystem. They are looking for feedback from users or projects that might have used or are about to use their RPC endpoints, and welcome suggestions to improve their service and evolve the ZkSync ecosystem. They have recently updated their rate limit for all shared plans of ZkSync Nodes from 60 to 200 requests/second (RPS) as part of ongoing service enhancements.
